src/main/features/testing-support/routes/create-claim-draft.ts

```diff
@@ -32,7 +32,11 @@ export default express.Router()
       draft.document = new DraftClaim().deserialize(prepareClaimDraft(user.email))
       await new DraftService().save(draft, user.bearerToken)
 
-      await claimStoreClient.addRoleToUser(user, 'cmc-new-features-consent-given')
+      const userRoles: string[] = await claimStoreClient.retrieveUserRoles(user)
+
+      if (userRoles.length === 0 || !userRoles.includes('cmc-new-features-consent-given')) {
```

Comment:
this check isn't right, as it doesn't stop you adding the role if its consent not given
```
!roles.some(role => role.includes('cmc-new-features-consent'))
```